packages/relay/src/runServer.ts

```diff
@@ -59,23 +60,35 @@ function sanitizeJsonRpcResponse (response?: JsonRpcResponse): JsonRpcResponse |
   return clone
 }
 
+function initializeAlternateProviders (alternateEthereumNodeUrls: string[] = []): Web3ProviderBaseInterface[] {
+  const alternateProviders: Web3ProviderBaseInterface[] = []
+  for (const url of alternateEthereumNodeUrls) {
+    alternateProviders.push(
+      new Web3.providers.HttpProvider(url)
```

Comment:
I *think* HttpProvider as a "timeout" parameter. this way, we don't wait indefinitely for each request, but up to a timeout.
(the alternative is to wrap the call to "sendTransaction" with a setTimeout ...)